src/bot.js

```diff
@@ -224,7 +193,8 @@ class HangoutsChatBot extends Adapter {
 
 
   /** Invoked when Event is received from Hangouts Chat. */
-  onEventReceived(event, res) {
+  onEventReceived(req, res, done = ()=>{}) {
```

Comment:
This breaks Pub/Sub, which passes the event directly. Please change it back to taking in an event or change startPubsubClient to pass in a req object with the event as the body.

Reply:
I fixed it. Let me know what you think.